crypto/init.c

```diff
@@ -586,64 +456,9 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
 
 int OPENSSL_atexit(void (*handler)(void))
```

Comment:
So this is now just a wrapper around `atexit` - which is fine, except we apparently call `OPENSSL_atexit` ourselves internally from a few places - which means we re-introduce the `atexit` issue. Specifically in:

crypto/s390xcap.c
providers/implementations/rands/seeding/rand_unix.c
ssl/rio/rio_notifier.c